### ISSUE TITLE
🆕 Software Suggestion |  Surfshark VPN

### ISSUE THREAD

**Issue by specter78** [privacytools/privacytools.io#2340]
## Basic Information

**Name:** Surfshark
**Category:** VPN
**URL:** https://surfshark.com/

## Description

No-logs VPN based in British Virgin Island which recently underwent third-party security audit.

## Why I am making the suggestion

As far as I am able to understand, it meets all criteria for vpn specified by PT.

## My connection with the software

None

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by youdontneedtoknow22** [comment]
Apps are not open source, so no way it's going to be recommended.
Even if it went open source, their marketing is a little bit too much. VPN's don't protect your from "Malware & Phishing" as they say, at least not from Phishing.

**Comment by Dyrimon** [comment]
Along with already presented concerns,

- They use scare tactics to force people into buying their service like "Not Protected" when you use their [ip address check service](https://surfshark.com/what-is-my-ip). This is against the criteria of PTIO.
- Website contains Google ads, tracking and analytics 
- Numerous evidence of buying or trying to buying online content creators to shill their service.

TLDR: A very shady VPN company with questionable ethics which should **not** be recommended.
